packages/vscode-extension/src/handlers.ts

```diff
@@ -364,17 +364,30 @@ export async function createNewProjectHandler(args?: any[]): Promise<Result<any,
   ExtTelemetry.sendTelemetryEvent(TelemetryEvent.CreateProjectStart, getTriggerFromProperty(args));
   const result = await runCommand(Stage.create);
   if (result.isOk()) {
-    await updateAutoOpenGlobalKey(args);
-    await ExtTelemetry.dispose();
-    // after calling dispose(), let reder process to wait for a while instead of directly call "open folder"
-    // otherwise, the flush operation in dispose() will be interrupted due to shut down the render process.
-    setTimeout(() => {
-      commands.executeCommand("vscode.openFolder", result.value);
-    }, 2000);
+    await openFolder(result.value, args);
   }
   return result;
 }
 
+export async function initProjectHandler(args?: any[]): Promise<Result<any, FxError>> {
+  ExtTelemetry.sendTelemetryEvent(TelemetryEvent.InitProjectStart, getTriggerFromProperty(args));
+  const result = await runCommand(Stage.init);
+  if (result.isOk()) {
+    await openFolder(result.value, args);
```

Comment:
Just to confirm, is readme also included for initialized project?

Reply:
Yes, for current PM spec, `README.md` is included. You could check the spec here https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_git/teamsfx-specs?path=/mods/ga/incremental_dev.md&version=GBjunjiespec&_a=preview&anchor=scaffolded-files